COCOAPODS.md

```diff
@@ -1,150 +1,343 @@
+file for the project.
 
-The podspec generated includes a path to an output framework and script phases which automate building
-this framework during a build process of an Xcode project. Some fields of the podspec file can be
-configured using the `kotlin.cocoapods { ... }` code block.
+The `Podspec` file includes a path to an output framework and script phases which automate building
+this framework during a build process of an Xcode project. 
 
-<div class="sample" markdown="1" theme="idea" mode="kotlin">
+## Add dependencies on Pod libraries
 
-```kotlin
-// Apply plugins.
-plugins {
-    id("org.jetbrains.kotlin.multiplatform") version "1.3.30"
-    id("org.jetbrains.kotlin.native.cocoapods") version "1.3.30"
-}
+You can add dependencies between a Kotlin project and Pod libraries [stored in the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
+and [stored locally](#add-a-dependency-on-an-objective-c-pod-library-stored-locally).
 
-// CocoaPods requires the podspec to have a version.
-version = "1.0"
+[Complete initial configuration](#install-the-cocoapods-dependency-manager-and-plugin), and when you add a new dependency, 
+just re-import the project in IntelliJ IDEA. The new dependency will be added automatically. No additional 
+steps are required.
 
-kotlin {
-    cocoapods {
-        // Configure fields required by CocoaPods.
-        summary = "Some description for a Kotlin/Native module"
-        homepage = "Link to a Kotlin/Native module homepage"
+### Add a dependency on a Pod library from the CocoaPods repository
 
-        // The name of the produced framework can be changed.
-        // The name of the Gradle project is used here by default.
-        frameworkName = "my_framework"
+1. Add dependencies on a Pod library that you want to use from the CocoaPods repository with `pod()`  to `build.gradle.kts` 
+(`build.gradle`) of your project.
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            pod("AFNetworking", "~> 3.2.1")
+        }
     }
-}
-```
+    ```
+    
+    </div>
 
-</div>
+2. Re-import the project.
 
-The following podspec fields are required by CocoaPods:
-  - `version`
-  - `summary`
-  - `homepage`
+To use this dependency from Kotlin code, import a package `cocoapods.<library-name>`.
+In the example above, it's `cocoapods.AFNetworking`.
+
+### Add a dependency on an Objective-C Pod library stored locally
+
+1. Add a dependency on a Pod library stored locally with `pod()` to `build.gradle.kts` (`build.gradle`) of your
+ project.  
+As the third argument, specify the path to `Podspec` of the local Pod using `projectDir.resolve("..")`.  
+    > You can also add dependencies as subspecs.  
```

Comment:
Do I understand correctly that subspecs work only for locally-stored pods?

Reply:
No, this syntax is available also for remote pods.

Reply:
Then the document is kinda misleading here: subspecs are mentioned under "stored locally" section, and the example shows only subspecs of locally-stored pods.

Reply:
Agree

Reply:
Let's add a subspec example for remote pods as well then. I didn't realize that. Thanks! Will ask @zoldater for such an example.

Reply:
Remote Subspec dependency added in [KotlinWithLocalObjCPods exapmle](https://github.com/zoldater/KotlinWithLocalObjCPods/blob/master/kotlin-library/build.gradle.kts)

Reply:
Added a note and example to a procedure on adding a remote Pod. Updated the example with local pods - included a remote Pod added via a subspec as well